spec/controllers/sessions_controller_spec.rb

```diff
@@ -1,19 +1,31 @@
-require 'spec_helper'
+require "spec_helper"
 
 describe Clearance::SessionsController do
   it { should be_a Clearance::BaseController }
 
-  describe 'on GET to /sessions/new' do
-    before { get :new }
+  describe "on GET to #new" do
+    context "when a user is not signed in" do
+      before { get :new }
 
-    it { should respond_with(:success) }
-    it { should render_template(:new) }
-    it { should_not set_the_flash }
+      it { should respond_with(:success) }
+      it { should render_template(:new) }
+      it { should_not set_the_flash }
+    end
+
+    context "when a user is signed in" do
+      before do
+        sign_in
+        get :new
+      end
+
+      it { should redirect_to(Clearance.configuration.redirect_url) }
```

Comment:
We are asserting that the user should be redirected to the `redirect_url` regardless of what the methods in the controller are set to. This is the default and we want this test to fail if the code is changed that implements this fix.